bottom-navigator/src/main/java/com/pandora/bottomnavigator/BottomNavigator.kt

```diff
@@ -130,15 +130,31 @@ open class BottomNavigator internal constructor() : ViewModel() {
      * that might not be feasable. By setting detachable = false the fragment's view will be kept
      * and memory and just hidden view.
      */
-    open fun addFragment(fragment: Fragment, detachable: Boolean = true) {
-        addFragmentInternal(fragment, currentTab, detachable)
+    open fun addFragment(
+        fragment: Fragment,
+        detachable: Boolean = true,
+        enterAnim: Int = 0,
+        exitAnim: Int = 0,
+        popEnterAnim: Int = 0,
+        popExitAnim: Int = 0
```

Comment:
These could be tagged with `@AnimatorRes @AnimRes` to make sure people don't pass in the wrong type of ints.

Reply:
Good call, updated